test/display_test.py

```diff
@@ -398,29 +332,27 @@ def todo_test_gl_set_attribute(self):
         'iconify is only supported on some video drivers/platforms'
     )
     def test_iconify(self):
```

Comment:
why is this function being changed here? I remember having updated this unit test a few months ago, and now you are reverting it?